content/en/blog/2020/wasmhub-istio

```diff
@@ -0,0 +1,51 @@
+---
+title: Extended and Improved WebAssemblyHub to Bring the Power of WebASsembly to Envoy and Istio
+subtitle: A hub to build, publish, share and deploy WebAssembly Envoy repos.
+publishdate: 2020-03-12
+attribution: "Idit Levine (Solo.io)"
+keywords: [wasm,extensibility,alpha,performance,operator]
+---
+As users look to adopt Envoy-based infrastructure to help them solve challenges with microservices communication, they inevitably find themselves needing to customize some part of that infrastructure to fit within their organization’s constraints. [WebAssembly (Wasm)](https://webassembly.org/) has emerged as a safe, secure, and dynamic environment for platform extension.
+
+Last week's [announcement of Istio 1.5](https://istio.io/blog/2020/wasm-announce/), lays the foundation for  bringing WebAssembly to the popular Envoy proxy. At [Solo.io](https://solo.io), we have teamed up with Google and the Istio community to simplify the overall experience of creating, sharing, and deploying WebAssembly extensions to Envoy and Istio. We all remember how not-so-long-ago Google and others laid the foundation for containers, and Docker built a great user experience to make it consumable. Similarly, we are making Wasm consumable by building the best user experience for WebAssembly on Envoy.
+
+According to Sean Suchter, Director of Engineering for Istio and Anthos Service Mesh, “Powerful tooling is nothing without a great developer experience. This is one of the many reasons Google and the Istio community are so excited to collaborate with the team behind Solo.io, which created WebAssembly Hub, on this important project to deliver enhanced extensibility to Istio. “
+
+We began our efforts to provide a great developer experience for WebAssembly with Envoy back in December 2019 when we announced WebAssembly Hub. We created a workflow that allows developers to very quickly spin up a new WebAssembly project in C++ (we’re expanding this language choice, see below), build it using Bazel in Docker, and push it to an OCI-compliant registry. From there, operators had to  pull the module, and configure Envoy proxies themselves to load it from disk. We added beta support in Gloo, our API Gateway built on Envoy, to declaratively and dynamically load the module, but we wanted to bring the same effortless and secure  experience to other Envoy-based frameworks as well. 
```

Comment:
As this isn't on the Solo blog any more, careful when you say "we"

Reply:
yes - doing another sweep through the blog to make adjustments for phrasing